Cargo.toml

```diff
@@ -205,6 +205,8 @@ unicode-width.workspace = true
 url.workspace = true
 walkdir.workspace = true
 supports-unicode = "3.0.0"
+completest = "0.4.2"
+completest-pty = "0.5.3"
```

Comment:
`completest-pty` re-exports `completest` so we only need one dep which will avoid needing to keep these in sync

Reply:
These also should be defined in `workspace.dependencies` and sorted

Reply:
Thanks for your reminder!

Reply:
And pulled into `dev-dependencies`, rather than `dependencies`